scripts/day2/edge-release-helm-oci-artefacts.txt

```diff
@@ -1,8 +1,13 @@
-edge/akri-chart:0.12.20
-edge/akri-dashboard-extension-chart:1.0.0
-edge/cdi-chart:0.3.0
-edge/endpoint-copier-operator-chart:0.2.0
-edge/kubevirt-chart:0.3.0
-edge/kubevirt-dashboard-extension-chart:1.0.0
-edge/metal3-chart:0.7.3
-edge/metallb-chart:0.14.3
+edge/3.1/akri-chart:0.12.20
```

Comment:
Should Elemental be part of this list?

Reply:
Rancher, Elemental and NeuVector images are shipped with the `rancher-images.txt` file that comes with each rancher [release](https://github.com/rancher/rancher/releases).